demos/dotnet/dotnet/Log.cs

```diff
@@ -4,39 +4,42 @@
 using System;
 using System.Collections.Generic;
 
-class Log
```

Comment:
Can we put this behind an interface? I don't imagine this is the only logger folks might want...